deployment/cloudways.rst

```diff
@@ -0,0 +1,158 @@
+.. index::
+   single: Deployment; Deploying to Cloudways Cloud Servers
+
+Deploying to Cloudways 
+======================
+
+Cloudways is a managed hosting provider for PHP based applications. This step by
+step article describes how to install Symfony on Cloudways servers and how to
+deploy Symfony applications using Git and SSH.
+
+Step 1: Prepare your server and application
```

Comment:
Should we really add the step numbers in the headlines? This will require a lot of work when steps are not necessary any more in the future or new steps are to be added.